tests/test_feed.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import pytest
+from datetime import datetime
+
+from cmsplugin_feed_ai.feed import get_feed_settings, _fetch_profile_feed
+from facebook import GraphAPIError
+from twython.exceptions import TwythonAuthError
+
+from cmsplugin_feed_ai.feed import FeedDisplayItem
+from test_data import facebook, twitter
```

Comment:
This package seems to be missing. Also, I'd appreciate it if these were imported with less ambiguous local names.

```python
from test_data import facebook as facebook_test_data
```

etc maybe?